react-demo-app/src/components/Softphone.tsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+import Card from "./Card";
+import { GuxButton } from "genesys-spark-components-react";
+import "./Softphone.css";
+import useSdk from "../hooks/useSdk";
+import PendingSessionsTable from "./PendingSessionsTable";
+import ActiveConversationsTable from "./ActiveConversationsTable";
+import HandledPendingSessionsTable from "./HandledPendingSessionsTable";
+import StationDetails from "./StationDetails";
+
+export default function Softphone() {
+  const [phoneNumber, setPhoneNumber] = useState("*86");
+  const [onQueueStatus, setOnQueueStatus] = useState(false);
+  const { startSoftphoneSession, updateOnQueueStatus } = useSdk();
+
+  function placeCall() {
+    startSoftphoneSession(phoneNumber);
+  }
+  function toggleQueueStatus() {
+    updateOnQueueStatus(!onQueueStatus);
+    setOnQueueStatus(!onQueueStatus);
+  }
+
+  // GuxButton's don't support form events currently so a form can't be used.
```

Comment:
This is incredibly aggravating because its a pretty core feature of HTML forms. I originally had this as a form but... gux was implemented and alas.